ip_tools.py

```diff
@@ -412,7 +412,7 @@ def full_scan_up(self, devices=None):
         
             self.spinner.text = f'Full-scan on {stylize(device.ip, recool.STYLE_HIGHLIGHT)} ({count - len(queue)}/{count})'
             self.interrupt_msg = f'Full-scan on {stylize(device.ip, recool.STYLE_HIGHLIGHT)} ({count - len(queue)}/{count})'
-            result = self.scan([device.ip], ['-A', '-p-', '-sV', '-Pn'], self.full_scan_sh)
+            result = self.scan([device.ip], ['-A', '-p-', '-sV', '-Pn' '--script=ssh-hostkey'], self.full_scan_sh)
```

Comment:
```suggestion
            result = self.scan([device.ip], ['-A', '-p-', '-sV', '-Pn', '--script=ssh-hostkey'], self.full_scan_sh)
```

shouldn't there be a coma?